ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Transaction.java

```diff
@@ -151,6 +213,35 @@ public Transaction(
     this.chainId = chainId;
   }
 
+  /**
+   * Instantiates a transaction instance.
+   *
+   * @param nonce the nonce
+   * @param gasPrice the gas price
+   * @param gasLimit the gas limit
+   * @param to the transaction recipient
+   * @param value the value being transferred to the recipient
+   * @param signature the signature
+   * @param payload the payload
+   * @param sender the transaction sender
+   * @param chainId the chain id to apply the transaction to
+   *     <p>The {@code to} will be an {@code Optional.empty()} for a contract creation transaction;
+   *     otherwise it should contain an address.
+   *     <p>The {@code chainId} must be greater than 0 to be applied to a specific chain; otherwise
+   */
```

Comment:
missing `it will default to any chain.` from original decleration.

Reply:
My bad

Reply:
Fixed